money-monkey-web/app/ui/MoneyField.tsx

```
@@ -0,0 +1,46 @@
+        autoComplete="off"
+        onFocus={() => {
+          field.onChange({
+            target: {
+              id: field.name,
+              value: field.value.replace(/[^0-9.]/g, "").replace(".00", ""), // remove all non-numeric characters
+            },
+          });
+        }}
+        onBlur={(e) => {
+          if (e.target.value.match(unformattedMoneyRegex)) {
+            e.target.value = usdFormatter.format(parseFloat(e.target.value));
```

Biome

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)